src/main/java/de/butbetter/keksys/CalculationController.java

```diff
@@ -0,0 +1,53 @@
+package de.butbetter.keksys;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("calc/v1/")
```

Comment:
Are you have any error with the requestmapping?

Reply:
no but i thought it would crash, if it doesnt nvm

Reply:
Explain what do you mean nvm(Non-Volatile Memory) ?

Reply:
nvm = never mind ^^

Reply:
I've not seen any error coming for there YET, seems like Spring Boot just doesn't care
I'd just ignore it
 
we won't deal with that side of the application anyways, since all the building and handling requests from a browser directly is being done on the Spring Boot Side, and if it can handle it
it's fine